vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -273,9 +273,13 @@ type internal FSharpProjectOptionsManager
         }
 
     member this.UpdateProjectInfoWithProjectId(projectId:ProjectId, userOpName) =
```

Comment:
I wonder if we should systematically clarify between "LegacyProject" and "ModernProject" throughout the naming of method like this, so 

    UpdateProjectInfoWithProjectId --> UpdateModernProjectInfoWithProjectId 
    SyncProject --> SyncLegacyProject

etc.  That may help us reason correctly about the two separate paths